test/models/user_test.rb

```diff
@@ -0,0 +1,61 @@
+require "test_helper"
+
+describe User do
+  let(:new_user) {
+    User.new(username: "test user")
+  }
+
+  it "can be instantiated" do
+    # Assert
+    expect(new_user.valid?).must_equal true
+  end
+
+  it "will have the required fields" do
+    # Arrange
+    new_user.save
+    user = User.first
+
+    # Assert
+    expect(user).must_respond_to :username
+  end
+
+  describe "validations" do
+    it "must have a username" do
+      new_user.username = nil
+
+      expect(new_user.valid?).must_equal false
+      expect(new_user.errors.messages).must_include :username
+      expect(new_user.errors.messages[:username]).must_equal ["can't be blank"]
+    end
+  end
+
+  describe "relationships" do
+    before do
+      new_user.save
+      @work_1 = Work.create(category: "book", title: "test title", creator: "test creator", publication_year: 2020, description: "test description")
+      @work_2 = Work.create(category: "album", title: "test title", creator: "test creator", publication_year: 2020, description: "test description")
+
+      Vote.create(user_id: new_user.id, work_id: @work_1.id)
+      Vote.create(user_id: new_user.id, work_id: @work_2.id)
```

Comment:
I recommend using `create!` in your `before` blocks so that you get an exception if it is invalid instead of a silent failure.